src/Compilers/Core/Portable/Operations/IHasDynamicArgumentsExpression.cs

```diff
@@ -11,29 +11,14 @@ namespace Microsoft.CodeAnalysis.Semantics
     public interface IHasDynamicArgumentsExpression : IOperation
     {
         /// <summary>
-        /// Name of the dynamically invoked member.
-        /// </summary>
-        string Name { get; }
-
-        /// <summary>
-        /// List of applicable symbols that are dynamically bound to the <see cref="Name"/>.
+        /// List of applicable symbols that are dynamically bound.
         /// </summary>
         ImmutableArray<ISymbol> ApplicableSymbols { get; }
```

Comment:
>ImmutableArray<ISymbol> ApplicableSymbols { get; } [](start = 8, length = 50)

I believe we decided to not expose any applicable symbols. In any case, given the name of the interface, it feels like this property doesn't belong in it.

Reply:
If we remove the property, is there a value in having this interface at all?

---
In reply to: [137881777](https://github.com/dotnet/roslyn/pull/21711#discussion_r137881777) [](ancestors = 137881777)

Reply:
I have removed the interface and moved the Arguments property to each IDynamicXXXExpression interface.